pom.xml

```diff
@@ -103,6 +103,7 @@
         <module>worker-store-s3</module>
         <module>worker-store-mem</module>
         <module>worker-tracking-report</module>
+        <module>woker-testing</module>
```

Comment:
Did you mean to add this?

Reply:
sorry Andy, will be careful from now on. I missed this. Will remove it.